src/core/sys/dragonflybsd/pthread_np.d

```diff
@@ -1,45 +0,0 @@
-/**
- * D header file for DragonFlyBSD
- *
- * Authors: Martin Nowak,Diederik de Groot(port:DragonFlyBSD)
- * Copied:  From core/sys/freebsd/sys
- */
-module core.sys.dragonflybsd.pthread;
```

Comment:
Why remove this file if it's there upstream?

Reply:
I can add it back in (but it is not used in the ltsmaster for now.
I would have to change the either the filename or the modulename. Which one do you prefer ?

Reply:
The name doesn't have to match, just leave it alone without a revert.  If you want to fix it, fix it upstream.

Reply:
Ok... 
Note: I did get an error when running the (ninja) unittests (which this file does not even contain), about pthread_np not existing.

Reply:
ninja warning or error?  It can spit a warning for a file that's not in the CMake file, not a big deal.

Are you saying this file is used upstream in master but not in this branch?  A grep turned up no use in master either.

Reply:
Actually an error (if i remember correctly, would have to rerun to check).
Link: https://github.com/dlang/druntime/pull/2107

Reply:
Nope doesn't seem to be used in master yet either (copied over from the freebsd implementation, but does not seem to be required. Maybe we should remove it altogetehr (in retrospect)).

Reply:
dlang#2107 got merged upstream.